grails-app/domain/com/asinha/domain/Payment.groovy

```diff
@@ -0,0 +1,36 @@
+package com.asinha.domain
+
+import com.asinha.base.BaseEntity
+import com.asinha.domain.Customer
+import com.asinha.domain.Payer
+import com.asinha.enums.PaymentMethod
+import com.asinha.enums.PaymentStatus
+
+class Payment extends BaseEntity {
+
+    BigDecimal value
+
+    String description
+
+    PaymentMethod method
+
+    PaymentStatus status
+
+    Date dueDate
+
+    Date paymentDate
+
+    Payer payer
+
+    Customer customer
+
+    static constraints = {
+        value min: 5.00, nullable: false
+        method nullable: false
```

Comment:
Nullable false é o default, não precisa especificar, só quando for nullable true